src/hooks/internal/useField.js

```diff
@@ -1,8 +1,13 @@
 import { useEffect } from "react";
 import useForm from "./useForm";
 import { getPath } from "../../utils/objectPath";
+import { ERROR_STRINGS } from "../../utils/error";
 
 function useField({ name, disabled, optional, validate, data }) {
+  if (!name || name.trim() === "") {
```

Comment:
Need to check for string here before trimming. Otherwise, as users type `<Input name` they'll get:

> name.trim is not a function

I'd do:

```
if (typeof name !== "string" || name.trim() === "")
```